### PR TITLE
Feature: Cookie SameSite directive support

### LINKED ISSUE #3: SameSite=None in SwooleEmitter.php
Hi,
emitCookies() in SwooleEmitter.php is missing the SameSite attribute, $cookie->getSameSite(). [Chrome 80](https://www.chromium.org/updates/same-site/incompatible-clients) which will be released on the 4th of february won't send cookies to a thirdparty without SameSite=None anymore. Also $cookie->getSameSite() returns a string like "SameSite= $value" which is also a problem. It should only return the value.

### PR DESCRIPTION
**Pull request by michalbundyra**
|    Q          |   A
|-------------- | ------
| New Feature   | yes

### Description

Updated ext-swoole dependency to 4.4.6 (as it is the first version,
where SameSite directive has been introduced).

Resolves #3 

References:
https://github.com/swoole/swoole-src/pull/2815/files
https://github.com/swoole/swoole-src/releases/tag/v4.4.6

### CODE REVIEW

**Comment by michalbundyra** [comment]
I think we have to drop `dflydev/fig-cookies` 1.0 support and, because of that, also PHP 7.1 support, as this version requires PHP 7.2 and only this version has support for SameSite cookie directive. Thoughts?

File: test/SwooleEmitterTest.php [hunk 1]
```diff
@@ -72,7 +72,7 @@ public function testMultipleSetCookieHeaders()
             ->withAddedHeader('Set-Cookie', 'bar=baz')
             ->withAddedHeader(
                 'Set-Cookie',
-                'baz=qux; Domain=somecompany.co.uk; Path=/; Expires=Wed, 09 Jun 2021 10:18:14 GMT; Secure; HttpOnly'
+                'baz=qux; Domain=somecompany.co.uk; Path=/; Expires=Wed, 09 Jun 2021 10:18:14 GMT; Secure; HttpOnly; SameSite=strict'
```
Comment (Ocramius) [inline test/SwooleEmitterTest.php]: Do we have tests for `'strict'`, `'Strict'`, `'lax'`, `'Lax'` and `''`? Or not relevant?

Comment (michalbundyra) [reply test/SwooleEmitterTest.php]: Good point, we do not have tests for all cases so far. I'll add these.

**Comment by michalbundyra** [comment]
@Ocramius thanks for review, but it can't be LGTM, as it does not work with `dflydev/fig-cookies` v1.0. So either we have:
- drop support of it and bump PHP version to 7.2
- add some check what version of `dflydev/fig-cookies` is used and skip SameSite at this case.

**Comment by Ocramius** [comment]
TBH, PHP 7.1 is no longer a thing.

**Pull request merged**